src/module/Ui/templates/discussion/discussions.twig

```diff
@@ -8,7 +8,7 @@
     <div class="panel-body">
         <ul class="list-group discussions">
             {% if isGranted('discussion.create') %}
-                {% set form = discussion().getForm('discussion', object, forum) %}
+                {% set form = discussion().getForm('discussion', object, forum, normalize().toUrl(entity)) %}
```

Comment:
Oh, wie kommt das denn da rein?

Ich glaub ich hatte an der Stelle ein bisschen rum experimentiert und vergessen das zu löschen.
Wenn ich nächstes mal da bin korrigier ich das auf jeden Fall.

Es könnte übrigens sein das die if-Abfrage die überprüft ob `$ref` null ist nicht richtig funktioniert. Entweder das oder `$url` zeigt auf die Artikel Seite in der die jeweilige Diskussion eingebettet ist und nicht auf die Diskussionsseite wie ich es erwartet hatte. Jedenfalls wird man in beiden Fällen dort hin geleitet, was aber auch nicht weiter schlimm ist.